app/models/verification/residence.rb

```diff
@@ -65,6 +66,10 @@ def store_failed_attempt
     })
   end
 
+  def adult?
+    (Date.today.year - self.date_of_birth.year) > 18
```

Comment:
Careful with this as one could be marked as an adult before (imagine someone that was born in the 31st December/ 2000).